juan-silva/tarea-3/spongeMeme.js

```diff
@@ -0,0 +1,14 @@
+function spongeMeme(sentence) {
+
+    let word = sentence.split('');
+    let meme = '';
+    for(let i = 0; i < word.length; i++){

```

Comment:
@jsilva97 , find the way to make this a one-like solution using modern approaches like forEach, map or reduce